cmd/bindings/bindings_test.go

```diff
@@ -164,7 +164,9 @@ func productTestServer() *httptest.Server {
 
 	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
-		json.NewEncoder(w).Encode(res)
+		if err := json.NewEncoder(w).Encode(res); err != nil {
+			return
```

Comment:
We always either return or log the error - unless the error is absolutely impossible or can't cause side-effects, in which case we can assign to _, but that should be rare. Apply this comment to the other cases...